packages/frontend/src/utils/date.ts

```diff
@@ -0,0 +1,16 @@
+import type { Dayjs } from "dayjs";
+import dayjs from "dayjs";
+
+export const getShortDate = (date: Dayjs) => {
```

Comment:
I'm guessing we could use the less error prone and more readable https://day.js.org/docs/en/durations/humanize, if I understand what this function does.

Reply:
I think we can use the `relativeTime` plugin, fixed in b83f121041f08ca64b4cbf12a249bb13742c846e.